src/App.jsx

```diff
@@ -1,20 +1,14 @@
 import { Layout, Menu } from 'antd'
-const { Header, Footer, Sider, Content } = Layout
+const { Header, Footer, Content } = Layout
+import { StyledSider } from './atoms.js'
 import './App.css'
 
 function App() {
   return (
   <Layout>
     <Header className='header'>Khinkalnya</Header>
```

Comment:
Please remove App.css completely by moving all styles to atoms.js
```suggestion
    <StyledHeader>Khinkalnya</StyledHeader >
```

Reply:
Fixed